app/views/GoldenTicketController/showVotesForProposal.scala.html

```diff
@@ -37,10 +37,10 @@ <h3>@Messages("gt.vote.ok")</h3>
                     }
 
                     <p>@Messages("gt.youcan")</p>
-                    <a class="btn btn-warning btn-sm" accesskey="r" href="@routes.GoldenTicketController.openForReview(proposal.id)" title="Shortcut : Ctrl-Option-r"><i class="icon-backward"></i>@Messages("gt.showVotes.review")</a><br>
-                    <a class="btn btn-primary btn-sm" accesskey="h" href="@routes.GoldenTicketController.showAllProposals()" title="Shortcut : Ctrl-Option-h"><i class="icon-home"></i> @Messages("gt.home")</a><br>
+                    <a class="btn btn-warning btn-sm" accesskey="r" href="@routes.GoldenTicketController.openForReview(proposal.id)" title="Shortcut : Ctrl-Option-r"><i class="fas fa-backward"></i>@Messages("gt.showVotes.review")</a><br>
+                    <a class="btn btn-primary btn-sm" accesskey="h" href="@routes.GoldenTicketController.showAllProposals()" title="Shortcut : Ctrl-Option-h"><i class="fas fa-home"></i> @Messages("gt.home")</a><br>
                     @if(nextToBeReviewedSameTrack.isEmpty && nextToBeReviewedSameFormat.isEmpty){
-                      <a class="btn btn-primary btn-sm" href="@routes.GoldenTicketController.showAllProposals()"><i class="icon-trophy"></i> @Messages("gt.nomore")</a><br>
+                      <a class="btn btn-primary btn-sm" href="@routes.GoldenTicketController.showAllProposals()"><i class="fas fa-trophy"></i> @Messages("gt.nomore")</a><br>
                     }
 
                   @if(nextToBeReviewedSameTrackAndFormat.nonEmpty && nextToBeReviewedSameTrackAndFormat.head.track.id == proposal.track.id) {
```

Comment:
@aheritier some `icon-forward` remains in the section below
FYI I fixed those in PR #193